QuestionAnswering/QA_ICL/frontend/README.md

```diff
@@ -1,4 +1,28 @@
-# Frontend Development for Marie
+# Marie Frontend
+
```

Comment:
Marie's frontend is only the Next app; it does not include the mock server. Please revise the heading.